content/document.js

```diff
@@ -599,7 +602,7 @@ function remove(
 
   if (dry) {
     if (redirectUrl) {
-      Redirect.add(locale, [[url, buildURL(locale, redirectUrl)]], { dry });
+      Redirect.add(locale, [[url, redirectUrl]], { dry });
```

Comment:
Not strictly related to what I intended to fix but it seemed to me that this was unecessary since `redirectURL` had already been built.

Reply:
I didn't realize that redirects can be made to external URLs, let alone absolute URLs -- apologies for breaking this in the first place, but thanks for fixing it up!

Reply:
That is yet another case for us documenting how the various parts of Yari works. This is something I am planing on doing very, very soon.